workflow/templateresolution/context.go

```diff
@@ -51,6 +51,13 @@ func WrapClusterWorkflowTemplateInterface(clusterClientset v1alpha1.ClusterWorkf
 	return &clusterWorkflowTemplateInterfaceWrapper{clientset: clusterClientset}
 }
 
+type NullClusterWorkflowTemplateGetter struct{}
+
+func (n *NullClusterWorkflowTemplateGetter) Get(name string) (*wfv1.ClusterWorkflowTemplate, error) {
+	return nil, errors.Errorf("", "invalid spec: clusterworkflowtemplates.argoproj.io `%s` is "+
```

Comment:
I think this error should be clearer: "Argo's ServiceAccount doesn't have RBAC access to ClusterWorkflowTemplates" should be enough

Reply:
This message is consistent with kubeclient error. In this scenario, the Context will be created using Informer. In some scenarios, context will be created using kubectl.